File: MosPosudit.UI/shared/lib/dtos/payment/paypal_order_response.dart
class PayPalOrderResponse {
  final String orderId;
  final String approvalUrl;
  final String status;

  PayPalOrderResponse({
    required this.orderId,
    required this.approvalUrl,
    required this.status,
  });

  factory PayPalOrderResponse.fromJson(Map<String, dynamic> json) =>
      PayPalOrderResponse(
        orderId: json['orderId'] ?? json['OrderId'] ?? '',
        approvalUrl: json['approvalUrl'] ?? json['ApprovalUrl'] ?? '',
        status: json['status'] ?? json['Status'] ?? '',
      );
}

File: MosPosudit.Model/Responses/Payment/PayPalOrderResponse.cs
namespace MosPosudit.Model.Responses.Payment
{
    public class PayPalOrderResponse
    {
        public string OrderId { get; set; } = string.Empty;
        public string ApprovalUrl { get; set; } = string.Empty;
        public string Status { get; set; } = string.Empty;
    }
}

